src/Utils/schedules.jl

```diff
@@ -7,7 +7,11 @@ false.
 """
 abstract type AbstractSchedule end
 
-initialize_schedule!(abstract_schedule) = nothing # fallback
+initialize_schedule!(schedule) = nothing # fallback
+
+# In general there is no alignment to be done.
+# We can only align if schedule::TimeInterval.
```

Comment:
It might be better not to add this comment, since it becomes incorrect when this function is overloaded. Perhaps better just to say "No alignment is default"?